docs/getting-started.md

```diff
@@ -70,13 +70,17 @@ From Source
 
     git clone https://github.com/django-json-api/django-rest-framework-json-api.git
     cd django-rest-framework-json-api
-    pip install -e .
```

Comment:
How does this work? I don't understand where the DJA package would be available to Python if it's not installed in editable mode.

Reply:
It's available because you are in the source tree for DJA so `import rest_framework_json_api` just works. I suppose retaining `pip install -e .` can't hurt. Pushing a new commit.
```
django-rest-framework-json-api$ tree -L 1
.
├── AUTHORS
├── CHANGELOG.md
├── LICENSE
├── MANIFEST.in
├── README.rst
├── docs
├── drf_example
├── env
├── example
├── manage.py
├── pytest.ini
├── requirements-development.txt
├── requirements.txt
├── rest_framework_json_api
├── setup.cfg
├── setup.py
└── tox.ini
```

Reply:
Sorry, neglected to answer the question: "Did you happen to run the steps you listed in Getting Started from a totally fresh clone to see if it would start the example app properly?"

Yes. Several times.